kubernetes/services/nginx-storage.yml.j2

```diff
@@ -29,15 +29,27 @@ spec:
             - mountPath:  /srv/storage/
               name:       concent-storage
             - mountPath:  /etc/nginx/conf.d/
-              name:       nginx-storage-config
+              name:       nginx-storage-configs
+              readOnly:   true
```

Comment:
You can't mount the whole map inside `conf.d/` because it contains `ssl.conf` which is not a standalone config. If you do this, nginx will always include `ssl.conf` at the top level. Put `ssl.conf` in a subdirectory (and make sure nginx does not load configs from subdirectories).

Reply:
OK. After talking with @bartoszbetka we're going to leave it as is. It's not included by the main config any more.

Reply:
Adding `ssl.conf` in `conf.d` folder causes that all configs in `http` directive depend on it. We use only one config with `server` directive, so there is no problem, to keep `ssl.conf` in `conf.d/` folder.